src/Copula.jl

```diff
@@ -31,6 +31,18 @@ function τ(C::Copula)
     r = Distributions.expectation(F,C; nsamples=10^4)
     return 4*r-1
 end
+function β(C::Copula{d}) where {d}

```

Comment:
So this beta does have a different deffinition in univariate and multivariate versions. I feel like we have to correctly define \tau and \rho when d > 2, the current definitions are wrong (and then if you want to itau and irho for some multivaraite models, you need the right definition)